src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/FlowRulesSpec.groovy

```diff
@@ -742,9 +742,10 @@ class FlowRulesSpec extends HealthCheckSpecification {
             [examVlanFlow.forward, examVlanFlow.reverse].eachParallel { direction ->
                 def resources = traffExam.startExam(direction)
                 direction.setResources(resources)
-                assert traffExam.waitExam(direction).hasTraffic()
+                assert traffExam.waitExam(direction)
```

Comment:
why change? It makes perfect sense to see traffic here so that later you expect packet counters growth

Reply:
As for me it allows us to be sure that flow is not rerouted while traffic flow is active.
refactored to:
```
assert !traffExam.isFinished(direction)
traffExam.waitExam(direction)
```